evo-delivery-app/evo-delivery-backend/.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules/
 .DS_Store
 dist/
+.env
+.idea/
```

Comment:
No need this line if it appears in root [.gitignore](https://github.com/tamirho/evo-delivery/pull/55/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947)